packages/flutter/lib/src/material/range_slider.dart

```diff
@@ -454,25 +454,25 @@ class _RangeSliderState extends State<RangeSlider> with TickerProviderStateMixin
   }
 
   // Finds closest thumb. If the thumbs are close to each other, no thumb is
-  // immediately selected while the drag velocity is zero. If the first
-  // non-zero velocity is negative, then the left thumb is selected, and if its
+  // immediately selected while the drag displacement is zero. If the first
+  // non-zero displacement is negative, then the left thumb is selected, and if its
   // positive, then the right thumb is selected.
   static final RangeThumbSelector _defaultRangeThumbSelector = (
       TextDirection textDirection,
       RangeValues values,
       double tapValue,
       Size thumbSize,
       Size trackSize,
-      double dx, // drag velocity
+      double dx, // drag displacement
```

Comment:
Maybe provide a little more explanation here. I assume that a drag's displacement is the current distance to the point where the down event occurred?

Reply:
Its the increment between updates